src/content/docs/dropins-b2b/purchase-order/containers/index.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Purchase Order Containers
+description: Overview of containers available in the Purchase Order drop-in.
+sidebar:
+  label: Overview
+  order: 1
+---
+
+import { Aside } from '@astrojs/starlight/components';
+import TableWrapper from '@components/TableWrapper.astro';
+
+The **Purchase Order** drop-in provides **12** pre-built container components for integrating into your storefront.
+
+<div style="background-color: var(--sl-color-blue-low); border-left: 4px solid var(--sl-color-blue); padding: 0.75rem 1rem; border-radius: 0.25rem; margin: 1rem 0;">
+<strong>Version: 1.0.0-beta2</strong>
+</div>
+
+## What are Containers?
+
+Containers are pre-built UI components that combine functionality, state management, and presentation. They provide a complete solution for specific features and can be customized through props, slots, and CSS.
+
+## Available Containers
+
+<TableWrapper nowrap={[0]}>
+
+| Container | Description |
+| --------- | ----------- |
+| [ApprovalRuleDetails](/dropins-b2b/purchase-order/containers/approval-rule-details/) | The ApprovalRuleDetails container displays and manages approval rule configuration including conditions, approvers, and rule metadata.. |
```

Comment:
Start each description with the verb instead of with "The X container"

Reply:
Yeah, that's a generator thing. I'll fix